### ISSUE TITLE
Outdated linux package (missing 1.5.2) in software.opensuse.org

### ISSUE THREAD

**Issue by Talkless** [qTox/qTox#3814]
I've noticed that there is 1.5.2 release, but my qTox installed  from software.opensuse.org is still 1.5.1:

```
apt policy qtox
qtox:
  Installed: 201609102219~v1.5.1
  Candidate: 201609102219~v1.5.1
  Version table:
 *** 201609102219~v1.5.1 500
        500 http://download.opensuse.org/repositories/home:/antonbatenev:/tox/xUbuntu_16.04  Packages
        100 /var/lib/dpkg/status
```

OS: Kubuntu 16.04 amd64
qTox version: 1.5.1-obs
Commit hash:   f05b7e040a073c7afadaf763dda5beaf130c6dbd
toxcore:  0.0.0
Qt:  5.5.1

**Comment by zetok** [comment]
It's not outdated, there's no difference between 1.5.1 and 1.5.2 for Linux.

**Comment by Talkless** [comment]
I believe fix for https://github.com/qTox/qTox/issues/1485 is missing. Or PR was too late for 1.5.2?

**Comment by zetok** [comment]
> I believe fix for #1485 is missing. Or PR was too late for 1.5.2?

Not really. 1.5.2 points to the same commit as 1.5.1 – new tag was needed for qTox re-release on windows with updated deps, since m$ broke video compatibility which made qTox 1.5.1 without patched ffmpeg crash.

**Comment by Talkless** [comment]
Oh I see. Thanks for explanation.
